CHANGELOG.md

```diff
@@ -1,9 +1,28 @@
-# dbt_tiktok_ads version.version
+# dbt_tiktok_ads v0.8.0
+
+[PR #24](https://github.com/fivetran/dbt_tiktok_ads/pull/24) includes the following updates:
+## Schema Changes
+**4 total changes • 0 possible breaking changes
+| Table/Column                                      | Change Type | Old Name | New Name                                  | Notes                                                             |
+|---------------------------------------------------|-------------|----------|-------------------------------------------|-------------------------------------------------------------------|
+| tiktok_ads__campaign_country_report       | New Model   |          |  | New table that represents the daily performance of a campaign at the country/geographic region level.               |
```

Comment:
Do you think we should we add links to these models in the DAG?

Reply:
Agreed - added